README.md

```diff
@@ -46,48 +46,34 @@ This required re-writing the history of the repo. You **must** [pull and rebase
 If you want to contribute to the code, please fork the repository, make your changes, and create a pull request.
 
 ### Local setup
+Build the necessary services:
 
-Install the [PostgreSQL](https://www.postgresql.org/) database server. It's a useful idea to setup [passwordless authentication for local connections](https://www.postgresql.org/docs/current/static/auth-methods.html#AUTH-TRUST).
+    sudo docker-compose build
```

Comment:
shouldn't use sudo, rather add user to the docker group

Reply:
Ok